web/source/resources/osk/kmwosk.css

```diff
@@ -434,35 +432,11 @@ div.android #keytip {background-color:#f00;}
 /* Static styles */
 
 .desktop-static.kmw-osk-inner-frame{margin:0;background-color:#666;border:2px solid #ad4a28;box-sizing:border-box;-moz-box-sizing:border-box;}
-.desktop-static .kmw-key-layer-group{width:100%;height:100%;margin:0;padding:0;border:none;background:transparent;}
-.desktop-static .kmw-key-layer{width:100%;height:100%;margin:0;padding:0;background-color:transparent;overflow:hidden;}
-.desktop-static .kmw-key-row{position:static;width:100%;height:20%;margin:0;padding:0;border:none;overflow:hidden;}        
```

Comment:
For all but the 'inner-frame' aspect and the `{position: relative}` bit for the `key-layer`, documentation keyboards now use the very same styling rules that their live equivalents do!